boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0.dts

```diff
@@ -69,11 +64,15 @@
 	status = "okay";
 };
 
-&i2c4 {
+&flexcomm0 {
```

Comment:
Seems like the place you set the flexcomm purpose is the same place you should set `status`

Reply:
My thinking here is that the board wiring dictates the flexcomm purpose, however we have multiple cores in this soc. Therefore I set the type in the common board dtsi, but enabled it only for core 0. An application might want to use an overlay to override the default and enable it in core 1 instead.

Reply:
that makes sense.